src/routes/safe/components/Apps/components/AppsList.tsx

```diff
@@ -19,6 +20,11 @@ const Wrapper = styled.div`
   flex-direction: column;
 `
 
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: ${fontColor};
```

Comment:
why is this needed? the link does not have any text and the color for the texts inside AppCard should be handled in there.

Reply:
good catch